helm/values.yaml

```diff
@@ -133,20 +141,16 @@ sideCar:
       CA_FILE_LOCATION: /opt/certs/ca/ca.pem
```

Comment:
Is that on use?

Reply:
I mean all of the ca/key/cert values

Reply:
comments due not in use right now - we will keep it just in case for future use

Reply:
done